src/main/java/uk/gov/companieshouse/api/testdata/service/impl/UserServiceImpl.java

```diff
@@ -105,15 +231,36 @@ public void updateUserWithOneLogin(String userId) {
             var existingUser = user.get();
             existingUser.setOneLoginUserId(userId);
             repository.save(existingUser);
+        } else {
+            LOG.debug("User not found for updateUserWithOneLogin with id= " + userId);
         }
     }
 
     @Override
     public Optional<User> getUserById(String userId) {
-        return repository.findById(userId);
+        LOG.debug("getUserById called for userId= " + userId);
+        var result = repository.findById(userId);
+        LOG.debug("getUserById found={}");
+        return result;
     }
 
     protected Instant getDateNow() {
         return LocalDateTime.now(ZONE_ID_UTC).toInstant(ZoneOffset.UTC);
     }
+
+    String generateIdentityId() {
```

Comment:
why do we need a method for this?  just put UUID.randomUUID().toString(); in line?